### DISCUSSION TITLE
Problem creating project in Google API Console in 2024

### DISCUSSION THREAD

**Discussion by Leodez** [jstedfast/MailKit#1696] [Q&A]
The instructions in the GMailOAuth2.md document to create the project in Google API Console no longer work that way. How can the project be created now? I really can't understand how to create the project with Google's changes. Especially since it is not going to relate to any domain.

**Comment by jstedfast** [comment]
Ugh, I hate that they keep changing their UI 😦 

It makes my life difficult trying to keep my docs updated to reflect their docs/UI.

Can you post a screenshot of what it looks like when you go to https://cloud.google.com/console ?

My guess is that you've already created a project and the screen has this in the upper-left corner (instead of **Select a project**):

![Screenshot 2024-01-31 104310](https://github.com/jstedfast/MailKit/assets/338984/ad31a859-4704-4c01-88b0-e0ffec5c3c31)

(Note: The name of the project in the drop-down menu there might be named something different for you, depending on what you named your project(s) in the past).

In the example above, you'd click **Google Play Console Developer** which will open a modal dialog that allows you to select a different project or create a new one.

**Reply by Leodez** [reply]
This is how I see it:

![image](https://github.com/jstedfast/MailKit/assets/34223072/8c1a8775-6dfd-416b-b71d-55d29cd454aa)

**Reply by jstedfast** [reply]
Click **SyS-InfoMail** at the top, next to the **Google Cloud** text.

**Reply by Leodez** [reply]
After clicking on sysinfo:
![image](https://github.com/jstedfast/MailKit/assets/34223072/019c0416-4e90-4fed-b245-8dd1e99a1d2a)

The problem is that when creating the project in Organization you must select either your organization or the "No organization" option. I understand that since it will not be a cloud solution but a desktop application that can be used by any user with any Google account, I must indicate "No organization", but then it asks me for "Location" and it does not let me select anything.

**Reply by jstedfast** [reply]
Doesn't it default to **No organization** by default? I guess I'm confused about the issue you are facing.

**Reply by Leodez** [reply]
Ok, I'll present the options so you can appreciate...

First option:
![image](https://github.com/jstedfast/MailKit/assets/34223072/bb0d3677-e01f-45e5-b06d-f31c73857c7c)

Second option:
![image](https://github.com/jstedfast/MailKit/assets/34223072/12948f25-2cd2-4296-a47d-b23095230d67)

When I click BROWSE:
![image](https://github.com/jstedfast/MailKit/assets/34223072/be7f5f0a-f7e1-4000-87ad-045a4e33384a)

I don't know which is the correct option and if I indicate "No organization" I cannot indicate "Location".

**Reply by jstedfast** [reply]
For me, I don't have an **Organization** section that I can even specify. I only see **Project name** and **Location** which, for me, defaults to **No organization**.

![Screenshot 2024-01-31 133141](https://github.com/jstedfast/MailKit/assets/338984/79d5c856-023f-43d3-acf3-646677c9d5f4)

**Reply by Leodez** [reply]
Ok, my problem is that I am creating it from an organization account. You are possibly creating it from an @gmail.com account

Then the question would be how to configure it in an organization.

It is very hard to understand Google.

**Reply by jstedfast** [reply]
Did you manage to figure out what you needed to do?
